src/gui/tray/TalkReplyTextField.qml

```diff
@@ -52,8 +48,8 @@ Item {
 
             icon {
                 source: "image://svgimage-custom-color/send.svg" + "/" + Style.menuBorder
-                width: 38
-                height: 38
+                width: Style.talkReplyTextFieldPreferredWidth * 0.15
```

Comment:
Are we sure we want the icon sized larger than the button itself?

Reply:
> Are we sure we want the icon sized larger than the button itself?

I don't know, see the screenshot. @jancborchardt?

Reply:
I have change it to the same size as the button.